prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -248,6 +248,25 @@ gke_backup_job_template: &gke_backup_job_template
           memory: 200Mi
           cpu: 80m
 
+aks_upgrade_job_template: &aks_upgrade_job_template
+  skip_report: true # set to true until job is stable
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/test-infra/kyma-cluster-infra:v20190528-8897828
+      securityContext:
+        privileged: true
+      command:
+      - "bash"
+      args:
+      - "-c"
+      - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/cluster-integration/kyma-aks-upgrade.sh"
```

Comment:
I think we are missing `INPUT_CLUSTER_NAME` variable definition. It would be good to name clusters based on standard prefix and randomised suffix. What do you think?

I believe `preset-creds-aks-kyma-integration` is also necessary.